src/Symfony/Component/Process/Process.php

```diff
@@ -1123,7 +1123,7 @@ public function getInput()
      *
      * This content will be passed to the underlying process standard input.
      *
-     * @param string|resource|\Traversable|self|null $input The content
+     * @param resource|self|string|\Traversable|null $input The content
```

Comment:
Same, could "self"  have a special rule ... ? :)

Reply:
Currently, Fixer does not support such logic.
Feel free to introduce it via [raising PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/compare)